JS CI is failing in the Admin UI lint check

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

The JS-CI admin is failing after this commit https://github.com/keycloak/keycloak/commit/a2275c189970e805cd363de729dad043773ca1ac. You can see the failure in this run for example:

https://github.com/keycloak/keycloak/actions/runs/25780884446/job/75747958647

```
2026-05-13T08:48:54.8884416Z ##[group]Run pnpm --fail-if-no-match --filter keycloak-admin-ui lint
2026-05-13T08:48:54.8884918Z [36;1mpnpm --fail-if-no-match --filter keycloak-admin-ui lint[0m
2026-05-13T08:48:54.8906771Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2026-05-13T08:48:54.8907095Z env:
2026-05-13T08:48:54.8907381Z   MAVEN_ARGS: -B -nsu -Daether.connector.http.connectionMaxTtl=25
2026-05-13T08:48:54.8907747Z   WORKSPACE: keycloak-admin-ui
2026-05-13T08:48:54.8907988Z ##[endgroup]
2026-05-13T08:48:55.2312044Z 
2026-05-13T08:48:55.2313469Z > @keycloak/keycloak-admin-ui@999.0.0-SNAPSHOT lint /home/runner/work/keycloak/keycloak/js/apps/admin-ui
2026-05-13T08:48:55.2314546Z > wireit
2026-05-13T08:48:55.2314757Z 
2026-05-13T08:48:55.3301068Z Analyzing
2026-05-13T08:48:56.3301329Z   0% [0 / 4] [1 running] ../../libs/keycloak-admin-client:generate:openapi
2026-05-13T08:48:59.3309740Z  25% [1 / 4] [1 running] ../../libs/keycloak-admin-client:build
2026-05-13T08:49:01.3322557Z  50% [2 / 4] [1 running] ../../libs/ui-shared:build
2026-05-13T08:49:08.3436558Z  75% [3 / 4] [1 running] lint
2026-05-13T08:49:13.4447952Z [baseline-browser-mapping] The data in this module is over two months old.  To ensure accurate Baseline data, please update: `npm i baseline-browser-mapping@latest -D`
2026-05-13T08:49:58.7800938Z
...
2026-05-13T08:49:58.8718222Z 
2026-05-13T08:49:58.8718625Z ✖ 452 problems (4 errors, 448 warnings)
2026-05-13T08:49:58.8718812Z 
2026-05-13T08:49:59.2031602Z 
2026-05-13T08:49:59.2032700Z ❌ [lint] exited with exit code 1.
2026-05-13T08:49:59.2037718Z ❌ 1 script failed.
2026-05-13T08:49:59.2226619Z /home/runner/work/keycloak/keycloak/js/apps/admin-ui:
2026-05-13T08:49:59.2229009Z  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @keycloak/keycloak-admin-ui@999.0.0-SNAPSHOT lint: `wireit`
2026-05-13T08:49:59.2229602Z Exit status 1
```

### Version

999.0.0-SNAPSHOT

### Regression

- [ ] The issue is a regression

### Expected behavior

lint should work

### Actual behavior

lint fails

### How to Reproduce?

cd /home/rmartinc/keycloak/js/apps/admin-ui
pnpm --fail-if-no-match

### Anything else?

_No response_

--- a/js/apps/admin-ui/src/clients/ClientDetails.tsx
+++ b/js/apps/admin-ui/src/clients/ClientDetails.tsx
@@ -246,7 +246,7 @@ export default function ClientDetails() {
   // crash on the first API call (the original bug from the review).
   const isSsfFeatureEnabled = isFeatureEnabled(Feature.Ssf);
   const isSsfRealmEnabled =
-    realmRepresentation?.attributes?.["ssf.transmitterEnabled"] === "true";
+    realmRepresentation.attributes?.["ssf.transmitterEnabled"] === "true";
   const showSsfTab =
     isSsfFeatureEnabled &&
     isSsfRealmEnabled &&

--- a/js/apps/admin-ui/src/clients/add/CapabilityConfig.tsx
+++ b/js/apps/admin-ui/src/clients/add/CapabilityConfig.tsx
@@ -59,7 +59,7 @@ export const CapabilityConfig = ({
   // that crashes on its first API call.
   const showSsfReceiverToggle =
     isFeatureEnabled(Feature.Ssf) &&
-    realmRepresentation?.attributes?.["ssf.transmitterEnabled"] === "true";
+    realmRepresentation.attributes?.["ssf.transmitterEnabled"] === "true";
   return (
     <FormAccess
       isHorizontal

--- a/js/apps/admin-ui/src/clients/ssf/tabs/EventSearchTab.tsx
+++ b/js/apps/admin-ui/src/clients/ssf/tabs/EventSearchTab.tsx
@@ -176,7 +176,7 @@ export const EventSearchTab = ({ client }: EventSearchTabProps) => {
               type="button"
               variant="primary"
               onClick={() => handlePendingLookup()}
-              isDisabled={pendingActionLoading || !pendingLookupJti?.trim()}
+              isDisabled={pendingActionLoading || !pendingLookupJti.trim()}
               data-testid="ssfPendingLookup"
             >
               {t("ssfPendingLookup")}

--- a/js/apps/admin-ui/src/clients/ssf/tabs/StreamTab.tsx
+++ b/js/apps/admin-ui/src/clients/ssf/tabs/StreamTab.tsx
@@ -178,9 +178,7 @@ export const StreamTab = ({
   const [createStreamEventsFilter, setCreateStreamEventsFilter] = useState("");
   const [createStreamProfile, setCreateStreamProfile] = useState<
     "SSF_1_0" | "SSE_CAEP"
-  >(
-    (client.attributes?.["ssf.profile"] as "SSF_1_0" | "SSE_CAEP") || "SSF_1_0",
-  );
+  >(client.attributes?.["ssf.profile"] as "SSF_1_0" | "SSE_CAEP");
   const [createStreamDescription, setCreateStreamDescription] = useState("");
   const [createStreamEventsOpen, setCreateStreamEventsOpen] = useState(false);
   const [createStreamSubmitting, setCreateStreamSubmitting] = useState(false);

